moped-editor/src/views/projects/projectView/ProjectTeamTable.js

```diff
@@ -321,14 +304,14 @@ const ProjectTeamTable = ({ projectId }) => {
             }
           },
         }}
-        data={makeListOfActivePersonnel(data.moped_proj_personnel)}
+        data={personnel}
         title={
           <Typography variant="h2" color="primary">
             Project team
           </Typography>
         }
         options={{
-          ...(data.moped_proj_personnel.length < PAGING_DEFAULT_COUNT + 1 && {
+          ...(data?.moped_proj_personnel?.length < PAGING_DEFAULT_COUNT + 1 && {
```

Comment:
I don't think this length should be different from the `personnel` Array, but just in case, would you consider checking pagination on `personnel` instead?

Reply:
thanks for flagging this - i removed pagination entirely. this is the only content on the whole tab. if someone wants to add 100 team members, fine.